ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/query/BlockchainQueriesTest.java

```diff
@@ -400,7 +400,7 @@ public void getOmmerByBlockHashAndIndexShouldReturnEmptyWhenIndexIsOutOfRange()
 
   @Test
   public void getOmmerByBlockHashAndIndexShouldReturnExpectedOmmerHeader() {
-    final BlockchainWithData data = setupBlockchain(3);
+    final BlockchainWithData data = setupBlockchain(4);
```

Comment:
Since `BlockDataGenerator` changed, the blocks these tests tested had an empty ommer list. I bumped up the number of blocks to get a block with a non empty ommer list.